Docs(plugin): Add splitChunks.defaultSizeTypes (#4300)
webpack/webpack.js.org: src/content/plugins/split-chunks-plugin.md

Maximum number of parallel requests at an entry point.

### `splitChunks.defaultSizeTypes`

`[string] = 'javascript'`

Sets the size types which are used when a number is used for sizes.

### `splitChunks.minChunks`

`number = 1`

> Minimum number of chunks that must share a module before splitting.

Reviewer: ```suggestion
The minimum times must a module be shared among chunks before splitting.
```
I think this can be improved.
